components/monitoring/logging/staging/kustomization.yaml

```diff
@@ -3,28 +3,3 @@ kind: Kustomization
 resources:
 - ../base
 - ../base/external-secrets
-patches:
-  - path: monitoring-logging-cloudwatch-forwarder-path.yaml
-    target:
-      name: monitoring-logging-cloudwatch-forwarder
-      group: external-secrets.io
-      version: v1beta1
-      kind: ExternalSecret
-  - target:
-      group: external-secrets.io
-      version: v1beta1
-      kind: ExternalSecret
-      name: log-forwarder-splunk-rhtap-application-external-secret
-    patch: |
-      - op: replace
-        path: /spec/dataFrom/0/extract/key
-        value: staging/monitoring/logging/fluentd/splunk-forwarder-rhtap-staging-application
```

Comment:
This patch shouldn't be removed. 
It's for the Splunk-related log forwarders.

Reply:
ack, fixed